AUTHORS

```diff
@@ -1,3 +1,4 @@
+Tarik Dzanic <tdzanic@tamu.edu>
```

Comment:
Move down a line to maintain alphabetical order. (PS You can use git amend and force-push to update your PR without having to open a new one.)

Reply:
Are you okay to tweak this?

Reply:
I think I did it properly, can you confirm?